docs/user/security/authentication/index.asciidoc

```diff
@@ -293,6 +294,109 @@ xpack.security.authc.providers:
+xpack.security.authc.providers:
+  anonymous.anonymous1:
+    order: 0
+    credentials:
+      apiKey: "VnVhQ2ZHY0JDZGJrUW0tZTVhT3g6dWkybHAyYXhUTm1zeWFrdzl0dk5udw=="
+-----------------------------------------------
+
+The configuration snippet above uses an API key string that is the result of base64-encoding of `id` and `api_key` fields returned from the {es} API joined by a colon. You can also specify these fields separately and {kib} will do concatenation and base64-encoding for you:
+
+[source,yaml]
+-----------------------------------------------
+xpack.security.authc.providers:
+  anonymous.anonymous1:
+    order: 0
+    credentials:
+      apiKey.id: "VuaCfGcBCdbkQm-e5aOx"
+      apiKey.key: "ui2lp2axTNmsyakw9tvNnw"
+-----------------------------------------------
+
+It's also possible to use {kib} anonymous access in conjunction with the {es} anonymous access.
+
+Prior to configuring {kib}, ensure that the anonymous access is enabled and properly configured in {es}. See {ref}/anonymous-access.html[Enabling anonymous access] for more information.
+
+Here is how your `kibana.yml` can look like if you want to use {es} anonymous access to impersonate anonymous users in {kib}:
+
+[source,yaml]
+-----------------------------------------------
+xpack.security.authc.providers:
+  anonymous.anonymous1:
+    order: 0
+    credentials: "elasticsearch_anonymous_user" <1>
+-----------------------------------------------
+
+<1> The `elasticsearch_anonymous_user` is just a special constant that doesn't have any other meaning
+
+[float]
+===== Anonymous access and other types of authentication
+
+You can also configure any other authentication mechanisms in addition to the anonymous access in {kib}. In this case Login Selector UI will present a dedicated `Continue as Guest` option for the anonymous access:
+
+[source,yaml]
+--------------------------------------------------------------------------------
+xpack.security.authc.providers:
+  basic.basic1:
+    order: 0
+  anonymous.anonymous1:
+    order: 1
+    credentials:
+      username: "anonymous_service_account"
+      password: "anonymous_service_account_password"
+--------------------------------------------------------------------------------
+
+[float]
+===== Anonymous access and embedding
+
+One of the most popular use cases for the anonymous access is when you embed {kib} into other applications and don't want to force your users to log in just to view it. If you configured {kib} to use anonymous access as a sole authentication mechanism, you don't need to do anything special while embedding {kib}. 
+
+The things become a bit tricky if you configure multiple authentication mechanisms and use Login Selector UI to log in to {kib}. In this case embedded {kib} doesn't know whether you want your users to use anonymous access or not. If the intention is to use anonymous access then you should add `auth_provider_hint=<anonymous-provider-name>` query string parameter to the {kib} URL.
```

Comment:
I don't think we need to qualify the Login Selector UI here. We are trying to discourage disabling this, so I think it's safe for us to assume in these docs that it'll be enabled.

```suggestion
If you have multiple authentication providers enabled, and you wish to automatically log in anonymous users when embedding, then you will need to add the `auth_provider_hint=<anonymous-provider-name>` query string parameter to the {kib} URL that you're embedding.
```

Reply:
> I don't think we need to qualify the Login Selector UI here. We are trying to discourage disabling this, so I think it's safe for us to assume in these docs that it'll be enabled.

Wholeheartedly agree!